src/test/run-pass/match-larger-const.rs

```diff
@@ -0,0 +1,12 @@
+
+#[derive(Eq, PartialEq)]
```

Comment:
this test works on the nightly playpen: http://play.integer32.com/?gist=550e66e34e4ba58a8e716f800d94520f&version=nightly&mode=debug

Reply:
The playpen is probably not on the latest nightly.

Reply:
ah indeed. sorry